### ISSUE TITLE
[appearance][accessibility][Dark Mode] Poor logo visibility in the README: low contrast

### ISSUE THREAD

**Issue by webknjaz** [pypa/pipx#1059]
**Describe the bug**

Having one of the dark themes enabled in the GitHub settings, the "pipx" part of the logo is barely in the repository README. I've got some screenshots to demonstrate this:

Dark dimmed theme:
![pipx-dark-mode-readme-logo-poor-constrast](https://github.com/pypa/pipx/assets/578543/f218a0fc-6d87-4c73-8218-2f31c6cf5401)

Dark default theme:
![pipx-dark-mode-readme-logo-poor-constrast-dark-default-theme](https://github.com/pypa/pipx/assets/578543/a810e900-c4cf-49b8-8fd1-232e99943f80)

**How to reproduce**

1. Enable Dark Mode on GitHub (https://github.com/settings/appearance)
2. Go to https://github.com/pypa/pipx
3. Scroll to the beginning of README just below the file list
4. See the left part of the pipx logo being hardly distinguishable from the background

**Expected behavior**

The "pipx" text part of the logo should have a good contrast ratio against the background. It should work with all the themes GitHub allows the end-users to have. As a bonus, it'd be nice to check it against the popular https://github.com/StylishThemes/GitHub-Dark user style too.

P.S. If somebody has a vector version of the logo, it'd be good to make use of that instead of a bitmap PNG — SVGs are well-supported in web environments nowadays.

**Comment by chrysle** [comment]
> P.S. If somebody has a vector version of the logo, it'd be good to make use of that instead of a bitmap PNG — SVGs are well-supported in web environments nowadays.

This has been asked for already in https://github.com/pypa/pipx/issues/76#issuecomment-1233230156, but there was no response. I could convert the image to SVG with GIMP.

**Comment by webknjaz** [comment]
Well, the bitmap part wouldn't look sharp, then. Only the text part can be remade in a scalable way with the proper font.

**Comment by gaborbernat** [comment]
PR welcome.

**Comment by gaborbernat** [comment]
@cs01 do you still have these resources please?

**Comment by chrysle** [comment]
I think the only images available are those in #76. The original author has been asked for a vector image, but not responded yet.

**Comment by dukecat0** [comment]
![logo](https://github.com/pypa/pipx/assets/68463158/e5c641af-b1d3-4717-96c1-3e8cbda51404)

I have an SVG generated from the original PNG file, and I think the quality isn't too bad?

**Comment by webknjaz** [comment]
It's much better, compared to the bitmap version, when I zoom in the web page on mobile. I'd say go for it. Including updating images in the docs. Maybe don't even keep the PNG in the repo

**Comment by gaborbernat** [comment]
I'm personally ok with that 👍
